ComputeAdmin/AzureStack.ComputeAdmin.psm1

```diff
@@ -469,15 +535,17 @@ function New-AzSServer2016VMImage {
                 }
 
                 Write-Verbose -Message "Making VHD bootable"
-                $null = Invoke-Expression -Command "$VHDDriveLetter`:\Windows\System32\bcdboot.exe $VHDDriveLetter`:\windows /s $VHDDriveLetter`: /f BIOS" -ErrorAction Stop
-            } catch {
+                $null = & "$VHDDriveLetter`:\Windows\System32\bcdboot.exe $VHDDriveLetter`:\windows /s $VHDDriveLetter`: /f BIOS" -ErrorAction Stop
```

Comment:
I think the syntax here should be & "xxx.exe" arg1 arg2
& "$VHDDriveLetter`:\Windows\System32\bcdboot.exe" "$VHDDriveLetter`:\windows" "/s" "$VHDDriveLetter`:" "/f" "BIOS"